### PR TITLE
Small changes to tilted bottom boundary layer example

### PR DESCRIPTION
**Pull request by tomchor**
This PR changes ~two things~ about the BBL example:

- The variable `κ` was used for both the von Karman constant and for the eddy diffusivity. Now it's just used for the von Karman constant
- ~When calling `NonhydrostaticModel` we were using ordered arguments, which could make a new user interpret that they should know the order of the arguments when calling `NonhydrostaticModel`. Now this call is made in a more usual way with keyword arguments, where the order doesn't matter.~

### CODE REVIEW

**Comment by navidcy** [comment]
I don't understand what you mean in the second point with "we were using ordered arguments"?

Syntax
```Julia
f(; x, y, z)
```
is the same as
```Julia
f(x=x, y=y, z=z)
```
and the order does not matter, e.g.
```Julia
julia> x=1; y=2; z=3;

julia> function f(; x=0, y=0, z=0)
           @show x
           @show y
           @show z
           return x+y+z
           end
f (generic function with 1 method)

julia> f(; x, y, z)
x = 1
y = 2
z = 3
6

julia> f(; y, z, x)
x = 1
y = 2
z = 3
6

```

File: examples/tilted_bottom_boundary_layer.jl [hunk 1]
```diff
@@ -128,10 +128,12 @@ v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
 # and a constant viscosity and diffusivity.
```
Comment (navidcy) [inline examples/tilted_bottom_boundary_layer.jl:128]:
```suggestion
# and a constant viscosity and diffusivity. Here we use a smallish value of ``10^{-4} m² s⁻¹``.
```

File: examples/tilted_bottom_boundary_layer.jl [hunk 2]
```diff
@@ -128,10 +128,12 @@ v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
...
 
 ν = 1e-4 # m² s⁻¹, small-ish
```
Comment (navidcy) [inline examples/tilted_bottom_boundary_layer.jl:130]:
```suggestion
```

File: examples/tilted_bottom_boundary_layer.jl [hunk 3]
```diff
@@ -128,10 +128,12 @@ v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
...
-κ = ν
-closure = ScalarDiffusivity(; ν, κ)
+closure = ScalarDiffusivity(ν=ν, κ=ν)
```
Comment (navidcy) [inline examples/tilted_bottom_boundary_layer.jl:131]:
```suggestion
closure = ScalarDiffusivity(ν=1e-4, κ=1e-4)
```

File: examples/tilted_bottom_boundary_layer.jl [hunk 4]
```diff
@@ -128,10 +128,12 @@ v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
...
 
-model = NonhydrostaticModel(; grid, buoyancy, coriolis, closure,
+model = NonhydrostaticModel(; grid,
+                            buoyancy = buoyancy,
+                            closure = closure,
+                            coriolis = coriolis,
```
Comment (navidcy) [inline examples/tilted_bottom_boundary_layer.jl:136]: I suggest we revert this...
I don't see what's the issue.
Anything after `;` is a `kwarg` and the order is not important.

Comment (tomchor) [reply examples/tilted_bottom_boundary_layer.jl]: Yes, sorry I misread what was there and thought the ordered arguments version was being used. Thanks for catching that

**Review by navidcy — APPROVED** [review]
let's wait for #2922

**Pull request merged**